packages/frontend/src/features/sqlRunner/store/cartesianChartBaseSlice.ts

```diff
@@ -206,5 +230,59 @@ export const cartesianChartConfigSlice = createSlice({
 
             config.display.stack = action.payload;
         },
+
+        setYAxisFormat: (
+            { config },
+            action: PayloadAction<{ format: string }>,
+        ) => {
+            if (!config?.display) return;
+
+            const validFormat = isFormat(action.payload.format)
+                ? action.payload.format
+                : undefined;
+
+            config.display.yAxis = config.display.yAxis || [];
+            config.display.yAxis[0] = {
+                ...config.display.yAxis[0],
+                format: validFormat,
+            };
+
+            // Update the format for series with yAxisIndex 0
+            if (config.display.series) {
+                Object.values(config.display.series).forEach((series) => {
+                    if (series.yAxisIndex === 0) {
```

Comment:
Why the 0? Im just not seeing it

Reply:
so that if you change the first series format, you also change the y axis format so that the label on the chart left is also _scaled_ to percentage. Hope that makes sense!